app/frontend/localStorage.ts

```diff
@@ -1,6 +1,6 @@
 const localStorageVars = {
   WELCOME: 'dontShowDisclaimer',
-  INFO_MODAL: 'dontShowInfoModal_14.07.2022',
+  INFO_MODAL: 'dontShowInfoModal_08.02.2022',
```

Comment:
The dates should be reversed
```suggestion
  INFO_MODAL: 'dontShowInfoModal_02.08.2022',
```
we use the date notation as in Slovak language.